loopd/utils.go

```diff
@@ -31,26 +31,40 @@ func getClient(cfg *Config, lnd *lndclient.LndServices) (*loop.Client,
 		MaxPaymentRetries:   cfg.MaxPaymentRetries,
 	}
 
-	// Now that we know where the database will live, we'll go ahead and
-	// open up the default implementation of it.
+	swapClient, cleanUp, err := loop.NewClient(
+		cfg.DataDir, swapDb, clientConfig,
+	)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	return swapClient, cleanUp, nil
+}
+
+func openDatabase(cfg *Config, network *chaincfg.Params) (
+	loopdb.SwapStore, *loopdb.BaseDB, error) {
```

Comment:
nit: linter also complains here about BaseDB not being used anywhere.

Reply:
This will be used by the PR that's building on this, so for now I've added `//nolint:unparam` which I will remove in my next commits.